CRM/Event/Form/Registration.php

```diff
@@ -643,18 +643,21 @@ public static function initEventFee(&$form, $eventID) {
     }
 
     //CRM-16456 get all price field including expired one.
-    $getAllPriceField = TRUE;
+    //CRM-20637 extend to backoffice registration form create mode
+    $validOnly = TRUE;
     $className = CRM_Utils_System::getClassName($form);
-    if ($className == 'CRM_Event_Form_ParticipantFeeSelection' && $form->_action == CRM_Core_Action::UPDATE) {
-      $getAllPriceField = FALSE;
+    if (($className == 'CRM_Event_Form_ParticipantFeeSelection' && $form->_action == CRM_Core_Action::UPDATE) ||
+      ($className == 'CRM_Event_Form_Participant')
+    ) {
+      $validOnly = FALSE;
```

Comment:
It looks like the name of the original variable `$getAllPriceFields` was misleading (inverted) but the new variable name is good.
Can we get a more meaningful comment added in here somewhere explaining the intent of the code? Eg. 
`// Allow selection of expired price-set options on backend forms`